test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
+
+
+  Scenario: routes can define reverse routes with 'as'
+    Given a file "routes.coffee" with the contents
+      """
+      module.exports = ({ GET }) ->
```

Comment:
That's a debatable point. I would say that's true if I give it a dedicated name through `as`. If the helper name is derived from the URL, and the helper just provides a safer way to encode URL parameters for example, then it seems plausible that it should change if the URL changes.